src/main/java/org/jabref/model/database/BibDatabaseContext.java

```diff
@@ -198,7 +198,13 @@ public boolean isBiblatexMode() {
 
         // 4. BIB file directory
         getDatabasePath().ifPresent(dbPath -> {
-            String parentDir = dbPath.getParent().toAbsolutePath().toString();
+            assert dbPath != null : "dbPath is null";
+            Path parentPath = dbPath.getParent();
+            if( parentPath == null ) {
+                parentPath = Paths.get(System.getProperty("user.dir"));
```

Comment:
Not sure whether that helps: The directory lookup is explained at https://github.com/JabRef/jabref/pull/459#issuecomment-161817966. The addition to help is still a TODO (https://github.com/JabRef/help.jabref.org/issues/160).

I also wait for @Siedlerchr s feedback 😇